application/common/common_functions.py

```diff
@@ -0,0 +1,119 @@
+# import os
```

Comment:
I don't think we need this file do we ? I think this is a temp file copied here from utils when we run the build . Possibly a hangover if you ran code locally . I'd remove